### PR TITLE
[README] Align to wording guidelines

### PR DESCRIPTION
**Pull request by mrackwitz**
/c @timanglade

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for more details!
 
 ## License
 
-Realm Cocoa is published under the Apache 2.0 license.  
+Realm Objective-C & Realm Swift is published under the Apache 2.0 license.
```
Comment (segiddins) [inline README.md]: Are published

**Comment by jpsim** [comment]
There are many more changes required to update the README for both bindings (9 links to `docs/cocoa`, among other things).

**Comment by mrackwitz** [comment]
@jpsim: Good call! I updated all the links and added where I thought it would make sense to have both also links to the Swift version.

**Comment by jpsim** [comment]
Could you update the links to use https while you're at it? Otherwise :+1:

**Comment by mrackwitz** [comment]
:+1:

**Comment by jpsim** [comment]
:+1: thanks

**Pull request merged**